app/src/main/res/layout/fragment_wallet.xml

```diff
@@ -43,16 +44,139 @@
 
                 <Button
                     android:id="@+id/sendButton"
-                    style="?android:attr/buttonBarButtonStyle"
+                    style="@style/ButtonStyle"
                     android:layout_width="wrap_content"
                     android:layout_height="match_parent"
                     android:layout_alignParentEnd="true"
                     android:layout_centerVertical="true"
-                    android:fontFamily="@font/lato_bold"
                     android:text="@string/button_send"
-                    android:textColor="@color/colorPrimary" />
+                    android:textSize="@dimen/text_size_medium"
+                    android:textColor="@color/blue3" />
+
             </RelativeLayout>
         </android.support.v7.widget.Toolbar>
 
     </android.support.design.widget.AppBarLayout>
-</LinearLayout>
\ No newline at end of file
+
+    <ScrollView
```

Comment:
This is OK for now (i.e temporary place holder) but we'll need to implement RecyclerView here so views can be reused and UI experience is much better for users. We can discuss more.

Reply:
Sounds good! 👍  Sorry I missed that in the commit message.